app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: {"data-turbo": "false"}) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  <div class="devise-form">
+    <div class="field">
+      <%= f.label :name %><br/>
+      <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+    </div>
+    <div class="field">
+      <%= f.label :email %><br/>
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+        <em>(<%= @minimum_password_length %>
+          characters minimum)</em>
+      <% end %><br/>
+      <%= f.password_field :password, autocomplete: "new-password" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password_confirmation %><br/>
+      <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+    </div>
+
+    <div class="actions">
+      <%= f.submit "Sign up", class:"devise-button" %>
+    </div>
+  </div>
+<% end %>
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] I am not able to register a new user please check your devise configuration and address it
<img width="1440" alt="Screen Shot 2022-12-07 at 21 35 20" src="https://user-images.githubusercontent.com/44745855/206279221-29c69c3b-d3a2-4018-b845-3b302c8b5481.png">